src/components/AclCheck.svelte

```diff
@@ -0,0 +1,34 @@
+<script type="ts">
+  import CircularProgress from "@smui/circular-progress";
+  import { onDestroy, onMount } from "svelte";
+
+  import Ethereum from "../pages/connect/odoo/Ethereum.svelte";
+  import { acl } from "../state/resolutions";
+
+  let timeout = null;
+  let displayConnect = false;
+
+  onMount(() => {
+    timeout = setTimeout(() => {
+      displayConnect = true;
+    }, 3000);
+  });
```

Comment:
`onMount` takes a function that can return an [unsubscribe function](https://svelte.dev/docs#run-time-svelte-onmount).

In that case you can do something like:

```typescript
  onMount(() => {
    timeout = setTimeout(() => {
      displayConnect = true;
    }, 3000);
    return () => clearTimeout(timeout);
  });
```

If you do so, remember to remove the `onDestroy` few lines from here.